src/Symfony/Component/Debug/FatalErrorHandler/UndefinedMethodFatalErrorHandler.php

```diff
@@ -36,8 +36,12 @@ public function handleError(array $error, FatalErrorException $exception)
 
         $message = sprintf('Attempted to call an undefined method named "%s" of class "%s".', $methodName, $className);
 
+        if (null === $methods = get_class_methods($className)) {
+            return new UndefinedMethodException($message, $exception);
+        }
+
         $candidates = array();
-        foreach (get_class_methods($className) as $definedMethodName) {
+        foreach ($methods as $definedMethodName) {
```

Comment:
:-1:
I think code should be as explicit as possible. Here we want to iterate methods if there are any. We don't want to cast the result to an array and iterate that and be depended on the result of the cast. To me it makes no sense that `(array) null` result in an empty array, I would expect an error. Another side effect of always casting is that it would be done even when not needed.